bundles/org.openhab.binding.netatmo/src/main/resources/ESH-INF/thing/camera.xml

```diff
@@ -24,7 +24,30 @@
 		</channels>
 
 		<representation-property>id</representation-property>
-		<config-description-ref uri="thing-type:netatmo:nawelcomecamera"/>
+		<config-description-ref uri="thing-type:netatmo:camera"/>
+	</thing-type>
+
+	<thing-type id="NOC">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="netatmoapi" />
+		</supported-bridge-type-refs>
+
+		<label>Presence Camera</label>
+		<description>This represents a presence camera at home</description>
+
+		<channels>
+			<channel id="cameraStatus" typeId="status"></channel>
+			<channel id="cameraSdStatus" typeId="sd_status"></channel>
+			<channel id="cameraAlimStatus" typeId="alim_status"></channel>
+			<channel id="cameraIsLocal" typeId="is_locale"></channel>
+			<channel id="cameraLivePicture" typeId="live_picture"></channel>
+			<channel id="cameraLivePictureUrl" typeId="live_picture_url"></channel>
+			<channel id="cameraLiveStreamUrl" typeId="live_stream_url"></channel>
+			<channel id="welcomeHomeEvent" typeId="homeEvent"></channel>
```

Comment:
Do we keep this channel with this name ?

Reply:
As I understand the events are reported to the WelcomeHome thing. Actually I haven't tested the channel and events of the Presence are useless at the moment, because sub events are missing within the retrofit Swagger mapping. I'm working on it separately as I mentioned in issue #3059. I think I will remove this channel for the moment and realize and test it with a separate issue later.

Reply:
Yes because that would have no sense to present a channel which is never set.

Reply:
done